### PR TITLE
Bug 1933184: Add maxUnavailable to DaemonSets

### PR DESCRIPTION
**Pull request by jsafrane**
To speed up cluster upgrade by allowing more nodes to be updated
simultaneously.

### CODE REVIEW

File: assets/node.yaml [hunk 1]
```diff
@@ -9,6 +9,10 @@ spec:
   selector:
     matchLabels:
       app: gcp-pd-csi-driver-node
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
```
Comment (wking) [inline assets/node.yaml:15]: `RollingUpdate` with `maxUnavailable: 1` is the default.  [The bug][1] is asking for `maxUnavailable: 10%`, so rollout time remains bounded as the node count grows.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1933184#c0

Comment (jsafrane) [reply assets/node.yaml]: Of course you're right. Fixed

**Comment by jsafrane** [comment]
gcp quota
/retest

**Comment by jsafrane** [comment]
/test e2e-gcp-csi
GCE API quota
(e2e-gcp installed and there are no errors related to storage)

**Pull request merged**